Platform/Ampere/JadePkg/PlatformPortingGuide.md

```diff
@@ -0,0 +1,372 @@
+# Ampere Altra Based Platforms Porting Guide
+
+## Overview
+
+The purpose of this document is to describe changes commonly made when adding a
+new Ampere Altra based platform.
+
+Users of this document should have prior firmware development experience with
+EDKII, including knowledge of UEFI specification.
+
+This document also assumes your new platform meets Ampere(R) Altra(R) Interface
+Firmware Requirements and is working with the Ampere system firmware and Trusted Firmware-A (TF-A).
+
+---
+
+## EDKII Code Organization
+
+The following diagram is the hierarchical structure of the silicon and platform
+directory. It is designed for easy portability and cross-platform reuse between
+Ampere Altra based platforms.
+
+```mermaid
+flowchart TD
+    edk2-platforms --> Silicon/Ampere
+    edk2-platforms --> Platform/Ampere
+    Silicon/Ampere --> AmpereSiliconPkg
+    Silicon/Ampere --> AmpereAltraPkg
+    Platform/Ampere --> JadePkg
+```
+
+All the packages are currently located under Silicon/Ampere and Platform/Ampere
+directories.
+
+The following table describes the intention of the above package organization:
+
+Directory Name    | Description
+------------------|-------------------------------------------------------------
+AmpereSiliconPkg  | Ampere silicon common code that can be shared for multiple silicon.
+AmpereAltraPkg    | Ampere Altra specific code.
+JadePkg           | Ampere Mt. Jade Platform code. Any new platform/board should be based on this package.
+
+## Build Instructions
+
+The build instructions can be found in [edk2-ampere-tools's README.md](https://github.com/AmpereComputing/edk2-ampere-tools)

```

Comment:
I guess we should get rid of this reference

Reply:
Agree. I will drop the Build Instructions section and contribute it later